src/main/java/embl/ebi/variation/eva/VariantJobsArgs.java

```diff
@@ -110,6 +117,7 @@
     private ObjectMap variantOptions  = new ObjectMap();
     private ObjectMap pipelineOptions  = new ObjectMap();
 
+    @PostConstruct
```

Comment:
It should not.
The init method (`loadArgs`) can be called multiple times (for instance if you want to refresh the config properties) `@PostConstruct` definition makes the warranty that the method is executed after creation and before injection.